android/src/main/java/com/swmansion/gesturehandler/react/RNGestureHandlerModule.kt

```diff
@@ -355,8 +352,20 @@ class RNGestureHandlerModule(reactContext: ReactApplicationContext?) :
     throw JSApplicationIllegalArgumentException("Invalid handler name $handlerName")
   }
 
-  @ReactMethod
```

Comment:
Are you sure you should remove the `@ReactMethod`? Will it work correctly on old arch?